packages/sui-widget-embedder/downloader/index.js

```diff
@@ -107,24 +107,38 @@
     })
   }
 
-  function matchPathnameWithRegExp(regExp) {
-    var pathname = window.location.pathname
+  function matchNameWithRegExp(name, regExp) {
+    if(!regExp) {
+      return false
+    }
     if (Array.isArray(regExp)) {
       return regExp.some(function(re) {
-        pathname.match(new RegExp(re))
+        name.match(new RegExp(re))
```

Comment:
i don't know how this was working before, you must return this value

Reply:
Good catch. It should be something like this:
`return regExp.some(re => name.match(new RegExp(re)))`

Reply:
Changed! I don't know but it was working